templates/admin/index.html

```diff
@@ -4,61 +4,53 @@
+        {{ render_project_card_option(project, upload_method, title=_('Subadmins'), explanation=_('Manage subadministrators'), link=url_for('admin.subadminusers'), link_action_text=_('Manage'), icon='user')}}
     </div>
-</div>
-<div class="row">
-    <div id="users-list" class="col-md-6">
+    <div id="users-list" class="col-md-4">
         {{ render_project_card_option(project, upload_method, title=_('Export Users'), explanation=_('Export a list of users'), link=url_for('admin.export_users', format='csv'), link_action_text=_('CSV'), link_two=url_for('admin.export_users', format='json'), link_action_text_two=_('JSON'), icon='user')}}
     </div>
-    <div id="import-users" class="col-md-6">
+    <div id="import-users" class="col-md-4">
         {{ render_project_card_option(project, upload_method, title=_('Import Users'), explanation=_('Import bulk users with local csv file'), link=url_for('admin.userimport', type='usercsvimport'), link_action_text=_('Import'), icon='user')}}
     </div>
-</div>
-{% endif %}
-<div class="row">
+    {% endif %}
     {% if current_user.admin %}
-    <div id="exportprojects" class="col-md-6">
+    <div id="exportprojects" class="col-md-4">
         {{ render_project_card_option(project, upload_method, title=_('Project Report'), explanation=_('Generate projects report'), link=url_for('project.export_projects', format='csv'), link_action_text=_('CSV'), icon='check') }}
     </div>
     {% endif %}
     {% if current_user.admin or current_user.subadmin %}
-    <div id="manageusers" class="col-md-6">
+    <div id="manageusers" class="col-md-4">
         {{ render_project_card_option(project, upload_method, title=_('Manage Users'), explanation=_('Enable/Disable Users'), link=url_for('admin.manageusers'), link_action_text=_('Manage'), icon='user')}}
     </div>
     {% endif %}
-</div>
-{% if current_user.admin %}
-<div class="row">
-    <div id="users" class="col-md-6">
-        {{ render_project_card_option(project, upload_method, title=_('Announcement'), explanation=_('Write new announcement'), link=url_for('admin.announcement'), link_action_text=_('Manage'), icon='user')}}
+    {% if current_user.admin %}
+    <div id="users" class="col-md-4">
+        {{ render_project_card_option(project, upload_method, title=_('Notices'), explanation=_('Write new announcement'), link=url_for('admin.announcement'), link_action_text=_('Manage'), icon='user')}}
     </div>
-    <div id="management-dashboard" class="col-md-6">
-        {{ render_project_card_option(project, upload_method, title=_('Management Dashboard'), explanation=_('Additional site-wide stats'), link=url_for('admin.management_dashboard'), link_action_text=_('Stats'), icon='info') }}
+    <div id="management-dashboard" class="col-md-4">
+        {{ render_project_card_option(project, upload_method, title=_('Stat Dashboard'), explanation=_('Additional site-wide stats'), link=url_for('admin.management_dashboard'), link_action_text=_('Stats'), icon='info') }}
```

Comment:
Not sure maybe Stats

Reply:
Yes :) https://github.com/bloomberg/pybossa-default-theme/pull/209/commits/df65ceeacc9974daf85fb3f0752d9b24c65b158d